.gitignore

```diff
@@ -27,66 +27,4 @@ hs_err_pid*
 .idea
 mvnw.cmd
 mvnw
-target
 
-##########################
-## Java
-##########################
-*.class
-.mtj.tmp/
-*.jar
-*.war
-*.ear
-hs_err_pid*
-
-##########################
-## Maven
-##########################
-target/
-pom.xml.tag
-pom.xml.releaseBackup
-pom.xml.versionsBackup
-pom.xml.next
-release.properties
-
-##########################
-## IntelliJ
-##########################
-*.iml
-.idea/
-*.ipr
-*.iws
-out/
-.idea_modules/
-
-##########################
-## Eclipse
-##########################
-.metadata
-.classpath
-.project
-.settings/
-bin/
-tmp/
-*.tmp
-*.bak
-*.swp
-*~.nib
-local.properties
-.loadpath
-
-##########################
-## NetBeans
-##########################
-nbproject/private/
-build/
-nbbuild/
-dist/
-nbdist/
-nbactions.xml
-nb-configuration.xml
-
-##########################
-## OS X
-##########################
-.DS_Store
```

Comment:
restore the full gitignore file and make a new additional commit

Reply:
thanks, I was repair gitinore, and commit it